Resources/Prototypes/Entities/Clothing/Shoes/magboots.yml

```
@@ -95,13 +95,13 @@
     toggleAction: ActionToggleMagbootsSyndie
   - type: ClothingSpeedModifier
     walkModifier: 0.95
-    sprintModifier: 0.9
+    sprintModifier: 1 #SS220 fix
     enabled: false
   - type: GasTank
-    outputPressure: 42.6
+    outputPressure: 21.2
```

```suggestion
    outputPressure: 42.6
```
42.6 - магическое число позволяющее дышать смесью азота и кислорода. Для чистого кислорода/азота хватает и офовского 21.2, но у тебя тут смесь 50 на 50.